src/script/conversation/EventMapper.ts

```diff
@@ -772,6 +765,16 @@ export class EventMapper {
     return messageEntity;
   }
 
+  /**
+   * Maps JSON data of E2E Identity verification message event to message entity.
+   */
+  private _mapEventE2EIVerificationMessage({data: eventData}: LegacyEventRecord): MissedMessage {
```

Comment:
```suggestion
  private _mapEventE2EIVerificationMessage({data: eventData}: E2EIDegradedMessageEvent | AllE2EIVerifiedEvent): MissedMessage {
```

This should work I believe and would provide better typing